Throwings.h

```diff
@@ -0,0 +1,51 @@
+﻿#pragma once
+
+#ifndef THROWINGS_H
+#define THROWINGS_H
+
+#include <SFML/Graphics.hpp>
+#include <SFML/Audio.hpp>
+#include <string>
+#include <cassert>
+#include <iostream>
+#include "ScoreAndLife.h"
+
+using namespace sf;
+using namespace std;
+
+class Throwings {
+private:
+    float x;
+    float speed;
+    Texture food_img;
+    Texture ddong_img;
+    Sprite food;
+    Sprite ddong;
+    Clock clock;
+    bool is_food;
+
+public:
+    bool hit = false;
+    bool destroyed = false;
+    bool destroyed_hit = false;
+    Throwings(float _x, float _speed, bool _is_food)
+        : x(_x), speed(_speed), is_food(_is_food)
+    {
+        if (this->is_food == true) {
+            assert(food_img.loadFromFile("img/food_img.png"));
+            food.setTexture(food_img);
+            food.setPosition(_x, 0.0f);
+        }
+        else if (this->is_food == false) {
```

Comment:
bool 타입이기에 true 혹은 false 둘 중 하나이므로 
```suggestion
        else  {
```
로 줄여 써주실 수 있을 것 같습니다!

Reply:
👍확인했습니다!